src/utils/getCalendarMonthWeeks.js

```diff
@@ -1,39 +1,39 @@
-export default function getCalendarMonthWeeks(month, enableOutsideDays) {
+import moment from 'moment';
+
+export default function getCalendarMonthWeeks(
+  month,
+  enableOutsideDays,
+  firstDayOfWeek = moment.localeData().firstDayOfWeek(),
```

Comment:
The propType should ensure that. I think that in general the utils functions should not assume safe inputs.